README.md

```diff
@@ -27,7 +27,7 @@ Then you can make a build for a specific target configuration e.g.
 
 The `TIMER_MAP` define sets up the timers default settings, as adjusted with the `timer` and `dma` commands in CLI. The problem is there are some magic numbers (index entries), that need to be known.
  
-You can activate the `timer map` CLI command (by compiling with `USE_TIMER_MAP_PRINT` defined i.e. `"EXTRA_FLAGS=-DUSE_TIMER_MAP_PRINT"`).
+You can activate the `timer map` CLI command (by compiling with `USE_TIMER_MAP_PRINT` defined i.e. `"EXTRA_FLAGS=-USE_TIMER_MAP_PRINT"`).
```

Comment:
This is intentional:

`-D[FLAG][=VALUE]` is a preprocessor flag.

Reply:
Sorry muy bad

Reply:
No worries - it's part of the make command.